code-challenges.js

```diff
@@ -7,62 +7,142 @@
+
+// const padres1984WorldSeriesRuns = [2, 5, 2, 2, 4]
+// const padres1998WorldSeriesRuns = [6, 3, 5, 3]
+// // Expected output: 9
 
-const padres1984WorldSeriesRuns = [2, 5, 2, 2, 4]
-const padres1998WorldSeriesRuns = [6, 3, 5, 3]
-// Expected output: 9
+// console.log(padres1984WorldSeriesRuns.concat(padres1998WorldSeriesRuns))
+// let padresBoth = padres1984WorldSeriesRuns.concat(padres1998WorldSeriesRuns)
+// console.log(padresBoth.length)
 
 // --------------------3) Create the code that will reverse the letters of a string using the test variable provided below.
 
 // Describe your process:
-
-const currentCohort = "Foxtrot 2022"
+// I split the directions into two sections which were splitting the letters (turing the string into an array), and I had to reverse the order.
+// I began with turning the string into an array, I then reversed the order of the array
+// After having the order that I wanted, I needed to turn the array back into a string
+// const currentCohort = "Foxtrot 2022"
 // Expected output: "2202 tortxoF"
+// let stringToArray = currentCohort.split("")
+// console.log(stringToArray.reverse())
+// console.log(stringToArray.join(""))
 
 // --------------------4) Create the code that will return the last index of the given value from the array using the test variables provided below.
 
 // Describe your process:
-
-const myNumbers = [13, 34, 42, 5, 5, 10, 27, 42, 10]
-const givenValue1 = 42
+// After looking at the array, I knew that the last time that 42 was displayed was index 7.
+// I then used .lastIndexOf() in order to get the exptected output.
+// const myNumbers = [13, 34, 42, 5, 5, 10, 27, 42, 10]
+// const givenValue1 = 42
 // Expected output: 7
+// console.log(myNumbers.lastIndexOf(42))
 
-const givenValue2 = 10
+// const givenValue2 = 10
 // Expected output: 8
+// console.log(myNumbers.lastIndexOf(10))
 
 // --------------------5) Create the code that will sort all the numbers in an array from largest to smallest. Use the test variables provided below.
 
 // Describe your process:
+// Knowing that I needed to sort out the array from least to greatest, I knew that I had to use the .sort() built in method.
+// Once I ran that on the terminal, I needed to reverse the order it was outputting.
+// I needed to go from least to greatest to greatest to least. 
+// I assumed that a-b would need to be reversed in order to get the array from largest to smallest.
 
-const sanDiegoSummerTemperatures = [79, 80, 72, 73, 82, 77, 76]
+// const sanDiegoSummerTemperatures = [79, 80, 72, 73, 82, 77, 76]
 // Expected output: [82, 80, 79, 77, 76, 73, 72]
+// sanDiegoSummerTemperatures.sort(((a, b) => b-a))
+// console.log(sanDiegoSummerTemperatures)
 
-const sanDiegoWinterTemperatures = [59, 68, 62, 59, 66, 67, 66]
+// const sanDiegoWinterTemperatures = [59, 68, 62, 59, 66, 67, 66]
 // Expected output: [68, 67, 66, 66, 62, 59, 59]
+// sanDiegoWinterTemperatures.sort(((a, b) => b-a))
+// console.log(sanDiegoWinterTemperatures)
```

Comment:
Outstanding job, Michy!  You have some of the best pseudocode I've seen and excellent javascript.  You don't have to copy the code for each given input -> you can just console.log them all at the bottom.  :) Great work!